geemap/coreutils.py

```diff
@@ -72,9 +72,9 @@ def ee_initialize(
     user_agent = f"{user_agent_prefix}/{__version__}"
     ee.data.setUserAgent(user_agent)
 
-    # pylint: disable-next=disable=protected-access
+    # pylint: disable-next=protected-access
     if ee.data._get_state().credentials is not None:
-        # pylint: enable-next=disable=protected-access
+        # pylint: enable-next=protected-access
```

Comment:
With disable-next, you do not need the enable. It only applies to the next line.